taxonomies/food/categories.txt

```diff
@@ -68225,6 +68238,31 @@ fr: Poires Conférence, Conférence
 hr: Konferencijske kruške
 nl: Conference peren
 
+< en:Pears
+en: Guyot pear
```

Comment:
```suggestion
en: Guyot pears
```

By convention all category names are in plural, with singular as a synonym (except if the synonym is just adding a final s, which should be supported automatically)

Reply:
(same for all the pears / poires below)